README.md

```diff
@@ -1 +1,12 @@
-# eos-icons-vsc
\ No newline at end of file
+# EOS Icons for VSCode
+
+This extension allows you to replace VSCode's default icons with those from the
+[EOS icon set](https://eos-icons.com/).
+
+
+## Usage
+
+Install the extension from the Marketplace, execute the command `Preferences:
+Product Icon Theme` and select the entry `EOS Icons` in the presented drop down:
+
```

Comment:
it would be great to add some more information about how or when to use the scripts defined in the package.json as it is not clear to me right now what I should do to test this PR for instance

Reply:
Sure, I'll add some documentation about these.